### DISCUSSION TITLE
What does the 'save' option under an generated image do?

### DISCUSSION THREAD

**Discussion by sraney** [Acly/krita-ai-diffusion#1014] [Q&A] [answered]
What does the save image option do and how does it save the image? Is there a location or is embedded inside the .kra file once more?  

![image](https://github.com/user-attachments/assets/a748d813-26e2-4cb5-b07b-c0de71c59349)

**Comment by syl1130** [comment]
only save image files, no .kra file

**Reply by sraney** [reply]
Where is that saved file then?

**Reply by Acly** [reply, marked as answer]
same location as the .kra document
